app/file.py

```diff
@@ -0,0 +1,45 @@
+from os import path
```

Comment:
`file` is a type in Python 2...Inasmuch as this is Py3 it is still not professional to call this file `file.py` because in the case of Py2 having `from app import file` would overwrite the builtin `file` type